src/components/FlowRunGraphSelectionPanel.vue

```diff
@@ -1,31 +1,36 @@
 <template>
-  <div v-if="node" class="flow-run-graph-selection-panel">
+  <div v-if="selection && isSelectionTaskOrFlow" class="flow-run-graph-selection-panel">
     <div class="flex justify-end">
       <p-button small icon="XMarkIcon" flat @click="closePanel" />
     </div>
-    <template v-if="node.kind === 'task-run'">
-      <FlowRunTimelineTaskDetails :task-run-id="node.id" />
+    <template v-if="selection.kind === 'task-run'">
+      <FlowRunTimelineTaskDetails :task-run-id="selection.id" />
     </template>
-    <template v-if="node.kind === 'flow-run'">
-      <FlowRunTimelineSubFlowRunDetails :flow-run-id="node.id" />
+    <template v-if="selection.kind === 'flow-run'">
+      <FlowRunTimelineSubFlowRunDetails :flow-run-id="selection.id" />
     </template>
   </div>
 </template>
 
 <script lang="ts" setup>
-  import { NodeSelection } from '@prefecthq/graphs'
+  import { GraphItemSelection, isNodeSelection } from '@prefecthq/graphs'
+  import { computed } from 'vue'
   import { FlowRunTimelineTaskDetails, FlowRunTimelineSubFlowRunDetails } from '@/components'
 
-  defineProps<{
-    node: NodeSelection | null,
+  const props = defineProps<{
+    selection: GraphItemSelection | null,
   }>()
 
+  const isSelectionTaskOrFlow = computed(() => {
+    return props.selection && isNodeSelection(props.selection)
+  })
```

Comment:
Seems like this condition should probably be external to this component. Probably don't even need to mount this component unless there is a selection and its is type `NodeSelection`.